Code/Source/solver/ComMod.h

```diff
@@ -1742,17 +1742,7 @@ class ComMod {
     /// @brief RIS mapping array, with global (total) enumeration
      std::vector<Array2D> grisMapList;
 
-    /// @brief Old time derivative of variables (acceleration); known result at current time step
-    Array<double>  Ao;
-
-    /// @brief New time derivative of variables (acceleration); unknown result at next time step
-    Array<double>  An;
-
-    /// @brief Old integrated variables (displacement)
-    Array<double>  Do;
-
-    /// @brief New integrated variables (displacement)
-    Array<double>  Dn;
+    /// @brief Ao, Do, Yo moved to Integrator class (complete ownership transfer)
```

Comment:
Remove comment: history is already preserved in `git`, no need to have in the code